examples/PythonProvider/python-flask-webapp-sample/app.py

```diff
@@ -0,0 +1,30 @@
+import os
+from flask import Flask, render_template, send_from_directory
+from azure.appconfiguration.provider import load_provider, AzureAppConfigurationKeyVaultOptions
+from azure.identity import DefaultAzureCredential
+
+app = Flask(__name__)
+
+ENDPOINT =  os.environ.get("AZURE_APPCONFIG_ENDPOINT")
+credential = DefaultAzureCredential()
+keyvault_options = AzureAppConfigurationKeyVaultOptions(credential=credential)
+azure_app_config = load_provider(endpoint=ENDPOINT, key_vault_options=keyvault_options, credential=credential)
```

Comment:
It will be good to add some comments explaining what this code is doing.

Reply:
Comments have been added.